runtime/turing/src/lib.rs

```diff
@@ -470,36 +472,11 @@ parameter_types! {
 	pub TemporaryForeignTreasuryAccount: AccountId = hex!["8acc2955e592588af0eeec40384bf3b498335ecc90df5e6980f0141e1314eb37"].into();
 }
 
-parameter_type_with_key! {
-	pub ExistentialDeposits: |currency_id: CurrencyId| -> Balance {
-		match currency_id {
-			CurrencyId::Native => EXISTENTIAL_DEPOSIT,
-			CurrencyId::UNIT => 10 * CurrencyId::UNIT.millicent(),
-			CurrencyId::KSM => 10 * CurrencyId::KSM.millicent(),
-			CurrencyId::AUSD => CurrencyId::AUSD.cent(),
-			CurrencyId::KAR => 10 * CurrencyId::KAR.cent(),
-			CurrencyId::LKSM => 50 * CurrencyId::LKSM.millicent(),
-			CurrencyId::HKO => 50 * CurrencyId::HKO.cent(),
-			CurrencyId::SKSM => 50 * CurrencyId::SKSM.millicent(),
-			CurrencyId::PHA => CurrencyId::PHA.cent(),
-		}
-	};
-}
-
+// TODO: remove after asset registry migration
```

Comment:
Can this be moved to the migration file? It doesn't look like it depends on any state in the runtime.